src/Functions/normalizeQuery.cpp

```diff
@@ -14,7 +14,7 @@ namespace ErrorCodes
     extern const int ILLEGAL_COLUMN;
 }
 
-namespace
+namespace NormalizeQuery
```

Comment:
After this change symbols become exported. Maybe leave anon namespace and rename symbols? (or make two nested namespaces but it will confuse reader).

Reply:
> After this change symbols become exported. Maybe leave anon namespace and rename symbols? (or make two nested namespaces but it will confuse reader).

Renaming is definitely a better choice. Thanks.